nixos/tests/boot.nix

```diff
@@ -104,6 +104,15 @@ in {
       bios = uefiBinary;
     };
 
+    directNetboot = makeTest {
+      name = "directboot-netboot";
+      nodes.machine = {
+        imports = [ ../modules/installer/netboot/netboot-minimal.nix ];
+        virtualisation.memorySize = 4096;
+      };
+      testScript = "machine.fail('stat /dev/vda')";
```

Comment:
We can probably do better than this. What about checking whether the nix store is mounted via a loop device? That's more of a key characteristic of netboot for me.